src/ui/qgisapp.ui

```diff
@@ -1886,7 +1886,7 @@
    </property>
    <property name="toolTip">
     <string>Moves a Label or Diagram
-Acts on all editable layers.</string>
+Acts on all layers.</string>
```

Comment:
What is the added value of leaving this message? Isn't it just stating the obvious and it would better be removed?

Reply:
Some of the labeling tools operate on the active layer only -- those also have an explanation in their tooltip reflecting this.

Reply:
We can also do that, but then we'll have to do it for all the feature tools etc. too I think?

Reply:
Sounds like a good idea in general to me!